web/styles/dark_theme.css

```diff
@@ -474,6 +474,31 @@
         color: hsl(236deg 33% 90%);
     }
 
+    #giphy_grid_in_popover {
+        .search-box {
+            display: flex;
+            position: sticky;
+            padding: 2px;
+
+            & #giphy-search-query {
+                background: hsl(225deg 6% 10%);
+                color: hsl(0deg 0% 75%);
+            }
+
+            & #giphy-search-query:focus {
+                background-color: hsl(225deg 6% 7%);
+                border: 1px solid hsl(0deg 0% 100% / 50%);
+                box-shadow: 0 0 5px hsl(0deg 0% 100% / 40%);
+            }
+        }
+    }
+
+    .emoji-popover-filter:focus {
+        background-color: hsl(225deg 6% 7%) !important;
+        border: 1px solid hsl(0deg 0% 100% / 50%) !important;
+        box-shadow: 0 0 5px hsl(0deg 0% 100% / 40%) !important;
+    }
```

Comment:
We are migrating styles away from `dark_theme.css`. Let's use variables for colors and use them `popovers.css`. Color variables are defined in `zulip.css`.

Reply:
Just a note that CSS variables are now defined in `app_variables.css`